web-components/src/components/stepper/index.tsx

```diff
@@ -132,6 +132,7 @@ const RegenStepper = ({
                   styles.alternativeLabel,
                   onStepClick && styles.hover,
                   classes?.stepLabel,
+                  'hover:cursor-default',
```

Comment:
this is overwritting the line above `onStepClick && styles.hover` 
I believe a better solution would be to have `onStepClick` not set at all for now in `StepperSection`